instrument/MARI_Definition.xml

```diff
@@ -4,9 +4,9 @@
 <instrument xmlns="http://www.mantidproject.org/IDF/1.0" 
             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
             xsi:schemaLocation="http://www.mantidproject.org/IDF/1.0 http://schema.mantidproject.org/IDF/1.0/IDFSchema.xsd"
- name="MARI" valid-from   ="1900-01-31 23:59:59"
-                        valid-to     ="2100-01-31 23:59:59"
-                        last-modified="2015-03-01 00:00:00">
+ name="MARI" valid-from   ="2016-09-11 23:59:59"
+                        valid-to     ="2100-01-01 23:59:59"
+                        last-modified="2016-11-02 15:50:00">
```

Comment:
Is this not supposed to be the date of last modification of the file?
Also I think `valid-to` is optional here.